server/src/main/java/org/candlepin/model/Pool.java

```diff
@@ -244,10 +245,12 @@ public String getDescription() {
     @ManyToOne
     @JoinColumn(name = "product_uuid", nullable = false)
     @NotNull
+    @JsonIgnore
     private Product product;
 
     @ManyToOne
     @JoinColumn(name = "derived_product_uuid")
+    @JsonIgnore
```

Comment:
This comment also applies to other places where JsonIgnore has been introduced on a field.